apps/pie-storybook/stories/pie-assistive-text.stories.ts

```diff
@@ -28,6 +28,13 @@ const assistiveTextStoryMeta: AssistiveTextStoryMeta = {
                 summary: defaultProps.variant,
             },
         },
+        isVisuallyHidden: {
+            description: 'If true, hides the component visually but leaves it accessible for a11y technologies.',
```

Comment:
```suggestion
            description: 'If true, hides the component visually but leaves it accessible for screen readers.',
```

This might be clearer, or saying 'accessibility' rather than 'ally' just in case it isn't as obvious :)